R/geom-curve.r

```diff
@@ -0,0 +1,64 @@
+#' Single curved line segments.
+#'
+#' @section Aesthetics:
+#' \Sexpr[results=rd,stage=build]{ggplot2:::rd_aesthetics("geom", "curve")}
+#'
+#' @inheritParams geom_point
+#' @param curvature see curveGrob
+#' @param angle see curveGrob
+#' @param ncp see curveGrob
+#' @param arrow specification for arrow heads, as created by arrow()
+#' @param lineend Line end style (round, butt, square)
+#' @seealso \code{\link{geom_segment}}, \code{\link{geom_path}} and \code{\link{geom_line}} for multi-
+#' segment lines and paths.
+#' @export
+#' @examples
+#' # Adding curve segments
+#' library(grid) # needed for arrow function
+#' b <- ggplot(mtcars, aes(wt, mpg)) + geom_point()
+#' b + geom_curve(aes(x = 2, y = 15, xend = 2, yend = 25, curvature = 0.2))
+#' b + geom_curve(aes(x = 2, y = 15, xend = 3, yend = 15, ncp = 2))
+#' b + geom_curve(aes(x = 5, y = 30, xend = 3.5, yend = 25), arrow = arrow(length = unit(0.5, "cm")))
+
+
+geom_curve <- function (mapping = NULL, data = NULL, stat = "identity",
```

Comment:
Can you please make curvature, angle, etc explicit arguments? I think it leads to better documentation.